src/middlewared/middlewared/migration/004_ntp_default_servers.py

```diff
@@ -0,0 +1,20 @@
+async def migrate(middleware):
```

Comment:
This file should be named `0003_ntp_default_servers_linux.py` because
- We only have 0001 and 0002, don't we?
- All other files are named with 4 digits, why be different here?
- If upgrading 11.3 -> 12.0 -> SCALE, this migration will be marked as executed in 12.0 and won't be ran again after upgrading to SCALE. If we name file `_linux` it will only be loaded on Linux which is what we desire.